src/constants/index.ts

```diff
@@ -1,16 +1,35 @@
-// Controller Response
+/* Controller Response */
+
 export const CREDENTIALS_INCORRECT = "Username or Password Incorrect";
 export const LOGIN_SUCCESSFUL = "Successfully logged in";
 export const USER_NOT_FOUND = "User not found";
-export const USERS_NOT_FOUND = "Users could not be found with the given arguements";
+export const USERS_NOT_FOUND =
+  "Users could not be found with the given arguements";
+export const GROUPS_NOT_FOUND =
+  "Groups could not be found with the given arguements";
+export const GROUP_NOT_FOUND = "Group not found";
+
+/* Validation */
 
-// Validation
+// Users
 export const LOGIN_FIELD_EMPTY = "Please enter a login";
+
 export const PASSWORD_FIELD_EMPTY = "Please enter a password";
-export const PASSWORD_PATTERN_INCORRECT = "Password must contain a letter and number"
-export const AGE_FIELD_EMTPY = "Please enter an age"
-export const AGE_PATTERN_INCORRECT = "Age must be between 4 and 130"
+export const PASSWORD_PATTERN_INCORRECT =
+  "Password must contain a letter and number";
+
+export const AGE_FIELD_EMTPY = "Please enter an age";
+export const AGE_PATTERN_INCORRECT = "Age must be between 4 and 130";
 
 // Auto Suggestions
 export const AUTO_SUGGESTIONS_SUBSTRING = "";
 export const AUTO_SUGGESTIONS_LIMIT = 10;
+
+// Groups
```

Comment:
Suggestion:
since constants are growing in numbers and it gets harder to group/separate them, you could start using object and nesting related constants in it.
Kind of like we did with I18Y in React Mentoring program.